tests/vmi_cloudinit_test.go

```diff
@@ -350,9 +350,12 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 		Context("with cloudInitNoCloud networkData", func() {
 			It("[test_id:3181]should have cloud-init network-config with NetworkData source", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdataNetworkData(
-					cd.ContainerDiskFor(cd.ContainerDiskCirros), "", testNetworkData, false)
-				LaunchVMI(vmi)
+				vmi := libvmi.NewCirros(
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
```

Comment:
Is it required to explicitly define a network and an interface in this test?
AFAIU, the test only checks the network config file.
Same below.

Reply:
`NewRandomVMIWithEphemeralDiskAndUserdataNetworkData` returned VMI with masquerade interface, therefore I followed.
If we set no interface, bridge interface will be added by default and it should work as expected.
Nevertheless, I think its better specifying the interface for better visibility, it may look odd for the next reviewer that thetest configure networking but no interface is specified.
Also, in case this default behavior miraculously change, this test would still be valid.